src/EPR.PRN.Backend.API/Repositories/Repository.cs

```diff
@@ -29,6 +31,30 @@ public async Task<List<Eprn>> GetAllPrnByOrganisationId(Guid orgId)
                         .ToListAsync();
         }
 
+        public async Task<List<PrnUpdateStatus>> GetModifiedPrnsbyDate(DateTime fromDate, DateTime toDate)
+        {
+            var result = await (from p in _eprContext.Prn
+                                join ps in _eprContext.PrnStatus on p.PrnStatusId equals ps.Id
+                                where p.LastUpdatedDate >= fromDate && p.LastUpdatedDate <= toDate
```

Comment:
During our dev discussion for this story we said we will go with Basic where condition as Suketu was away. Can we confirm where condition with him and if we need any changes we need to add ticket on backlog

Reply:
Discussed with Suketu he said we need to retrieve only accepted and rejected and date range should be on StatusUpdatedOn